cubano-httpeasy/src/main/java/org/concordion/cubano/driver/http/ProxySearch2.java

```diff
@@ -0,0 +1,87 @@
+package org.concordion.cubano.driver.http;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import java.net.ProxySelector;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Automatically detect proxy.
+ * 
+ * This does not support proxy automation script (PAC) which is a big show stopper so this class is a waste of time and I'm using proxy-vole instead. However
+ * proxy-vole (https://github.com/MarkusBernhardt/proxy-vole) seems like overkill and so this is remaining as a starting point for a lightweight implementation...
```

Comment:
Overkill from a user point of view, or implementation point of view? If this is only here for historical purposes, can be delete? It will always be in the Git history.

Reply:
Overkill from implementation point of view - the number of extra libraries it downloaded seemed overly large.

I've deleted the class